docs/content/guides/developer/getting-started.mdx

```diff
@@ -4,10 +4,15 @@ title: Getting Started
 
 Sui is the first internet-scale programmable blockchain platform. That might read like marketing speak, but as you peruse the documentation to understand the technology, you will discover that Sui addresses many of the problems that hold blockchains back from mass adoption. 
 
-Before you can get started developing on Sui, you need to understand the code repository and install its binaries. Consequently, the first two topics you need are:
+Before you can get started developing on Sui, you need to understand the code repository and install its binaries. 
 
-- [Sui Environment Setup](./getting-started/sui-environment.mdx)
-- [Install Sui](./getting-started/sui-install.mdx)
+## Install Sui 
+
+{@include: ../../snippets/quick-install.mdx}
+
+You can also install Sui from binaries or from source, as detailed in the Install Sui page.
+
+Go to [Install Sui](./getting-started/sui-install.mdx).
```

Comment:
```suggestion
```

Reply:
I like your suggestion better and is what I was actually going to do, but I changed it to match the format of all these other top level pages. UL for link lists of more than one item and separate line with "Go to LINK" for single links. We might want to revisit that structure at some point, but leaving for now to be matchy matchy.